bot/submission/submit.py

```diff
@@ -392,26 +412,34 @@ async def list_patterns(self, ctx: Context):
                 content="Here are the available patterns:", embed=utils.info_embed("Patterns", ", ".join(names))
             )
 
+
     @Cog.listener(name="on_raw_reaction_add")
```

Comment:
Did you ever notice that this listener is only for adding reactions

Reply:
Forgot that you're using auto reaction removal so this is actually fine. You can't cancel your vote though.